analyzers/src/SonarAnalyzer.CSharp/Rules/AvoidLambdaExpressionInLoopsInBlazor.cs

```diff
@@ -45,7 +45,7 @@ public sealed class AvoidLambdaExpressionInLoopsInBlazor : SonarDiagnosticAnalyz
                 {
                     var node = (LambdaExpressionSyntax)c.Node;
 
-                    if (IsWithinLoop(node)
+                    if (IsWithinBlockLoop(node)
```

Comment:
Suggestion:
- `IsWithinLoopBlock`
- `IsWithinLoopBody` (my favorite - it's not very clear what a "block of a loop" is)
- `IsWithinBlockOfLoop`